resources/assets/components/StatusButton/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './status-button.scss';
+// import { map } from 'lodash';
+
+
+class StatusButton extends React.Component {
+  getClass() {
+    return `button -${this.props.type}`;
```

Comment:
The [`classnames`](https://www.npmjs.com/package/classnames) helper is really nice for this!